templates/index.html

```diff
@@ -3,34 +3,64 @@
   <head>
     <title>Play Nim</title>
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
```

Comment:
An example of using AJAX with JQuery for Flask includes a line to also put the Google Library in the project static folder for 'fallback'. 
Do you think that might be useful here? 

The example is: 
```
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="{{
  url_for('static', filename='jquery.js') }}">\x3C/script>')</script>
```

Reply:
Yes, that's absolutely a good idea. We don't want our site to go down just because `ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js` did, but we still want to steal their bandwidth when we can.